FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -140,8 +146,25 @@ const GeneralInfo = (props) => {
     };
 
     const onUpdateField = e => {
+        const { value: fieldValue, name: fieldName } = e.target;
+        const symbolCount = (fieldValue.replace(/[\s`]/g,'').trim()).length;
+        setFormStateErr({ ...formStateErr, [fieldName]: {'error': false, 'message': ''}});
+        if (fieldName === 'name' && symbolCount < 2) {
+            setFormStateErr({ ...formStateErr, [fieldName]: {'error': true, 'message': 'Введіть від 2 до 100 символів'}});
+        }
+        if (fieldName === 'official_name' && symbolCount !== 0 && symbolCount < 2) {
+            setFormStateErr({ ...formStateErr, [fieldName]: {'error': true, 'message': 'Введіть від 2 до 200 символів'}});
+        }
         setProfile((prevState) => {
-            return { ...prevState, [e.target.name]: e.target.value };
+            return { ...prevState, [fieldName]: fieldValue };
+        });
+    };
+
+    const onBlurHandler = (e) => {
+        const { value: rawFieldValue, name: fieldName } = e.target;
+        const fieldValue = rawFieldValue.replace(/\s{2,}/g,' ').trim();
+        setProfile((prevState) => {
+            return { ...prevState, [fieldName]: fieldValue.trim() };
```

Comment:
The `.trim(`) method is called twice on `fieldValue`. It might be unnecessary

Reply:
Fixed